doc/book/app_server/luajit_memprof.rst

```diff
@@ -61,38 +61,40 @@ Below is a chunk of simple Lua code named ``test.lua`` to illustrate this.
     end
     local stp, err = misc.memprof.stop()
 
-Starting profiler in Lua code:
+The Lua code for starting the profiler -- as in line 3 in the test.lua example above -- is:
 
 ..  code-block:: lua
 
     local str, err = misc.memprof.start(FILENAME)
 
-where ``FILENAME`` is a name of the binary file where profiling events are written.
+where ``FILENAME`` is the name of the binary file where profiling events are written.
```

Comment:
@Buristan
profiling events = the events that are profiled? i.e. not the events that happen during profiling?

Reply:
Profiling events are described above. There are 3 possible events interesting for memory profiler: allocation, reallocation and deallocation.